scripts/tests.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from django.contrib.auth.models import User
+
+from expenses.models import Expenses
+from house.models import House
+from scripts.create_mock_data import run
+
+
+@pytest.mark.django_db()
+class TestScripts:
+    def test_create_mock_data(self):
+        mock_num_for_test = 10
+        run(mock_num_for_test)
```

Comment:
what does this command do?
Is this the way to run a script?

Reply:
Yes
this test makes sure that the create_mock_data script works
to run the script run: ... manage.py runscript <script file name>
you can look at the docs here: https://django-extensions.readthedocs.io/en/latest/runscript.html?highlight=script